guide/english/angular/services-and-injectors/index.md

```diff
@@ -2,6 +2,7 @@
 title: Services and Injectors
 ---
```

Comment:
I was stating that how about instead of creating a service manually with ng generate service [name],why can't the service be added automatically within the component just like the typescript,css and html files.